package.json

```diff
@@ -35,18 +39,19 @@
   },
   "homepage": "https://github.com/all-contributors/all-contributors-cli#readme",
   "dependencies": {
+    "@babel/runtime": "^7.2.0",
     "async": "^2.0.0-rc.1",
```

Comment:
How come we need to distribute a new dep?

Reply:
I find that `You should add @babel/runtime as dependency to your package` in ci error log.

refs: https://circleci.com/gh/all-contributors/all-contributors-cli/94?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
Ahh not sure how that works

Reply:
@jakebolam In my understanding of how `kcd-script build` works is that it uses `babel` under-the-hood so it a suggestion from somewhere on that level. Perhaps Kent could enlight us on that?